controller/linodemachine_controller.go

```diff
@@ -91,6 +91,7 @@ type LinodeMachineReconciler struct {
 	client.Client
 	Recorder         record.EventRecorder
 	LinodeApiKey     string
+	LinodeDNSApiKey  string
```

Comment:
nit: Capitalize initialisms and acronyms.

```suggestion
	LinodeDNSAPIKey  string
```

Reply:
Agree. But I kept it to match the field above `LinodeApiKey` (I'd not like to update both and also change the `LinodeApiKey` everywhere in the scope of this PR